files/class-api-cache.php

```diff
@@ -95,5 +120,10 @@ public function remove_file( $filepath ) {
 			unlink( $this->files[ $file_name ] );
 			unset( $this->files[ $file_name ] );
 		}
+
+		// Remove cached stats too if any
+		if ( isset( $this->file_stats[ $file_name ] ) ) {
```

Comment:
Technically we can omit the `isset()` check here for simplicity b/c PHP won't raise a warning if you `unset()` an undefined variable.

Reply:
Good point. Changed in 990f7cd84cc27773ecc94026cdb9773cf77821e8